src/ansys/fluent/core/launcher/error_handler.py

```diff
@@ -29,6 +29,14 @@ def __init__(self):
         super().__init__("Provide either 'ip' and 'port' or 'server_info_file_name'.")
 
 
+class IncorrectIpPortProvided(ValueError):
+    """Raised when IP address and port are incorrect."""
+
+    def __init__(self):
+        """Initialize IncorrectIpPortProvided."""
+        super().__init__("Provide correct 'ip' and 'port' or 'server_info_file_name'.")
```

Comment:
```suggestion
        super().__init__("Provide a valid 'ip' and 'port' or a 'server_info_file_name'.")
```
I'm also not sure if 'server_info_file_name' should be mentioned as a specific argument name like this because it could appear out of context (for instance if PyFluent is used in a higher-level workflow)?

Reply:
Ok. Removed.